eth/handler.go

```diff
@@ -452,6 +468,18 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 		// Filter out any explicitly requested headers, deliver the rest to the downloader
 		filter := len(headers) == 1
 		if filter {
+			// Check for any responses not matching our whitelist
+			if whitelist := pm.whitelist; whitelist != nil {
+				if expected, ok := (*whitelist)[headers[0].Number.Uint64()]; ok {
+					actual := headers[0].Hash()
+					if !bytes.Equal(expected.Bytes(), actual.Bytes()) {
+						p.Log().Debug("dropping peer with non-matching whitelist block", "number", headers[0].Number.Uint64(), "expected", expected, "actual", actual)
```

Comment:
Please capitalize log messages: `Dropping`

Instead of `actual` use `hash` and move it as the second context parameter after number.

Reply:
Done.